sale_order_type_user_default/models/sale_order.py

```diff
@@ -15,4 +15,5 @@ def _get_order_type(self):
 
     type_id = fields.Many2one(
         'sale.order.type',
-        default=_get_order_type)
+        default=_get_order_type,
```

Comment:
@ValentinoDefelice this default should be this way :
`  default=lambda so: so._get_order_type()`